examples/chef/common/chef-rvc-mode-delegate.cpp

```diff
@@ -197,14 +250,63 @@ void RvcCleanMode::Shutdown()
     }
 }
 
+chip::Protocols::InteractionModel::Status chefRvcCleanModeWriteCallback(chip::EndpointId endpointId, chip::ClusterId clusterId,
+                                                                        const EmberAfAttributeMetadata * attributeMetadata,
+                                                                        uint8_t * buffer)
+{
+    // this cluster is only enabled for endpoint 1
+    if (endpointId != 1)
+    {
+        return chip::Protocols::InteractionModel::Status::UnsupportedEndpoint;
+    }
+
+    VerifyOrDie(gRvcCleanModeInstance != nullptr);
+
+    chip::Protocols::InteractionModel::Status ret;
+    ModeBase::Instance * clusterInstance = gRvcCleanModeInstance;
+    chip::AttributeId attributeId        = attributeMetadata->attributeId;
+
+    switch (attributeId)
+    {
+    case chip::app::Clusters::RvcCleanMode::Attributes::CurrentMode::Id: {
+        uint8_t m                                  = static_cast<uint8_t>(buffer[0]);
+        Protocols::InteractionModel::Status status = clusterInstance->UpdateCurrentMode(m);
+        if (Protocols::InteractionModel::Status::Success == status)
+        {
+            break;
+        }
+        ret = chip::Protocols::InteractionModel::Status::UnsupportedWrite;
+        ChipLogError(DeviceLayer, "Invalid Attribute Update status: %d", static_cast<int>(status));
+    }
+    break;
+    default:
+        ret = chip::Protocols::InteractionModel::Status::UnsupportedAttribute;
+        ChipLogError(DeviceLayer, "Unsupported Attribute ID: %d", static_cast<int>(attributeId));
+        break;
+    }
+
+    return ret;
+}
+
+chip::Protocols::InteractionModel::Status chefRvcCleanModeReadCallback(chip::EndpointId endpointId, chip::ClusterId clusterId,
+                                                                       const EmberAfAttributeMetadata * attributeMetadata,
+                                                                       uint8_t * buffer, uint16_t maxReadLength)
+{
+    if (sizeof(*attributeMetadata) <= sizeof(buffer))
+    {
+        memcpy(buffer, &attributeMetadata, sizeof(attributeMetadata));
```

Comment:
This I do not understand either ....

Realistically if this is a "we do not care about the value" you can maybe 

```cpp
// NOTE: we do NOT care about a real implementation here at this point
memset(buffer, 0, maxReadLength)
```